src/kOS/Module/kOSProcessor.cs

```diff
@@ -240,11 +240,14 @@ public ModifierChangeWhen GetModuleCostChangeWhen()
             return ModifierChangeWhen.FIXED;
         }
 
+        public float additionalMass = 0F;
         private void UpdateCostAndMass()
         {
+            
             float spaceDelta = diskSpace - baseDiskSpace;
             additionalCost = (float)System.Math.Round(spaceDelta * diskSpaceCostFactor, 0);
             additionalMass = spaceDelta * diskSpaceMassFactor;
+            additionalMassGui = spaceDelta * diskSpaceMassFactor * 1000;
```

Comment:
Your totally wright. Thought of it after i posted the PR. Was waiting for your reaction to see if you wished to see it changed.